runners/suite_test.go

```diff
@@ -99,3 +105,21 @@ var _ = AfterSuite(func() {
 	err := testEnv.Stop()
 	Expect(err).ToNot(HaveOccurred())
 })
+
+func createStorageClass(ctx context.Context, name, provisioner string, enabled bool) {
```

Comment:
Because you give only true to `enabled` parameter, this can be removed.
Do you have any plans to use this?

Reply:
Thank you. I fixed it.
https://github.com/topolvm/pvc-autoresizer/pull/48/commits/2ee0accd1cc6a96260998c22409557801f949807